reconcile/rhidp/ocm_oidc_idp/standalone.py

```diff
@@ -99,6 +99,7 @@ def get_clusters(
             )
             for ocm_clusters in clusters_by_org.values()
             for c in ocm_clusters
+            if c.ocm_cluster.console
```

Comment:
@chassing i needed to change the `console` field in the `OCMCluster` to `Optional` because a HCP reports as ready while the `console` might still be missing for a while.

Reply:
It is ok for new clusters, but hopefully, we don't see it for existing ones!